en/docs/develop-components/sharing-and-reusing/use-a-connection-in-your-service.md

```diff
@@ -236,4 +236,43 @@ As the service URL you can use the URL that you resolved in [step 2](#step-2-rea
 
     !!! note
         If you want to consume a Choreo service at the project visibility level, you don't need to obtain a token. You can directly invoke the service using the resolved URL.
-        
+
+## Configure resiliency for the connection
+
+To ensure your service remains robust and responsive when consuming other Choreo services through connections, you can configure resiliency patterns. Choreo supports configuring retries for connections.
+
+    !!! note
+        Currently, the resiliency configuration is only available in the CDPs and only for paid users. It will be available for PDPs soon.
+
+### Configure retry
+
+To configure retry for your connection, follow these steps:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev).
+2. In the header, click the **Project** list. This opens the project home page.
+3. In the left navigation menu, click **Dependencies** and then click **Connections**.
+4. Click on a required **Connection** to view its details.
+5. Expand the **Advanced Configuration** section.
+6. Enable the **Retry** toggle under **Resiliency**.
+7. Set appropriate values for the retry parameters. For more information, see [Retry configuration parameters](#retry-configuration-parameters).
+8. Click **Save**.
+
+    !!! note
```

Comment:
Is the info in the note correct? I didn't see any info on the the `Choreo-API-Key` header in step 1.